api/src/main/java/org/openmrs/validator/ObsValidator.java

```diff
@@ -68,6 +69,7 @@ public boolean supports(Class<?> c) {
 	 * @should not validate if obs is voided
 	 * @should not validate a voided child obs
 	 * @should fail for a null object
+	 * @should fail for a differece in obs.getPerson and obs.getEncounter().getPatient()
```

Comment:
I think we moved away from @should annotation

Reply:
@wluyima I will delete line 72

Reply:
Are you deleting this?

Reply:
@wluyima I deleted line 72 and pushed in pull request 2291